manage.py

```diff
@@ -1,12 +1,22 @@
 #!/usr/bin/env python
+import os
 
 from coaster.manage import init_manager
 
 from imgee.models import db
 from imgee import app
 
 
+def mkdir_p(dirname):
+    if not os.path.exists(dirname):
+        os.makedirs(dirname)
+
 if __name__ == "__main__":
     db.init_app(app)
     manager = init_manager(app, db)
+
+    @manager.command
+    def init():
+        mkdir_p(os.path.join(app.project_root, app.config['UPLOADED_FILES_DEST']))
```

Comment:
Add `from coaster.manage import manager` at the top. `init_manager` just returns the same manager. You can discard its return value and use the imported manager.